### ISSUE TITLE
Docker Desktop (Windows) Install has multiple 500 Internal Server Error issues. 

### ISSUE THREAD

**Issue by trevor68** [janeczku/calibre-web#2735]
Installing Calibre-Web on Windows 11 using Docker Desktop (HyperV not WSL) and 6.2.0 (latest) is almost there but has a lot of different page errors, I presume due to some missing dependencies perhaps? 

I have installed COPS and got it all working, so I know that my books volume is working ok. I am pretty new to docker, and a windows guy, this VM runs on Truenas Scale and has operating caddy reverse proxy etc. 

I can grab more logs/screenshots as required. 
![admin](https://user-images.githubusercontent.com/91324171/229020130-9cfaca8d-6ce8-4274-a7e9-191936499a4f.jpg)
![500_error](https://user-images.githubusercontent.com/91324171/229020135-4453ff22-025f-405f-9c40-eedf8769a494.jpg)
[calibre-web.log](https://github.com/janeczku/calibre-web/files/11118053/calibre-web.log)

**Comment by OzzieIsaacs** [comment]
This error is the standard permissions not sufficient error for reading the metadata.db database

**Comment by OzzieIsaacs** [comment]
Here: https://github.com/janeczku/calibre-web/issues/2728, here a solutions seems to be given: https://github.com/janeczku/calibre-web/issues/2231

if this isn‘t helping please ask for help from the docker container provider

**Comment by trevor68** [comment]
Ah thanks, that will be why COPS is working then, I think it only reads the DB, I didn't realise CW has to write to it as well sorry. Though I did try to give it permission. I am attaching my docker compose and also how I created the volume in case I've done anything stupidly wrong there. I will go have a read of your suggestions now thanks. 
[createvolume.txt](https://github.com/janeczku/calibre-web/files/11118502/createvolume.txt)
[docker-compose.txt](https://github.com/janeczku/calibre-web/files/11118507/docker-compose.txt)
